corehq/apps/reports/dbaccessors.py

```diff
@@ -11,5 +12,19 @@ def get_exports(domain):
         "couchexport/saved_export_schemas",
         startkey=startkey,
         endkey=endkey,
-        include_docs=True,
+        include_docs=include_docs,
```

Comment:
I think danny was splitting it into two functions e.g. `get_exports_json` and `get_exports`